### PR TITLE
Expose HTTP error codes without panicking

### PR DESCRIPTION
**Pull request by auroter**
To test:

1) Do a full docker reset or make sure that your MetaFab wallet doesn't exist in Postgres
2) Create an organization and API key
3) Paste the API key into the Unity Demo GameLogicBehavior.cs
4) Run the Demo
5) Initialize MetaFab
6) Log In / Create User

Verify that a user is created.

This validates that the HTTP response of Login is being checked against 400, and that Create User is being called in response.

### CODE REVIEW

File: Runtime/StringWebView.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ async void Start()
             setup();
             // Load a URL once the weview has loaded
             await vuplexWeb.WaitUntilInitialized();
-            vuplexWeb.WebView.LoadUrl("https://payment-iframe.dev.string-api.xyz/index.html");
+            vuplexWeb.WebView.LoadUrl("https://payment-iframe.string-api.xyz/index.html");
```
Comment (saito-sv) [inline Runtime/StringWebView.cs]:
```suggestion
            vuplexWeb.WebView.LoadUrl("https://payment-iframe.string-api.xyz/");
```

File: Runtime/ApiClient.cs [hunk 2]
```diff
@@ -118,25 +118,25 @@ public async UniTask<T> Delete<T>(string path, object body = null, Dictionary<st
 		/// <param name="token">cancellation token</param>
 		/// <typeparam name="T">A <see cref="System.Serializable" /> class to populate with data. </typeparam>
 		/// <returns>T</returns>
-		public async UniTask<T> Patch<T>(string path, object body = null, Dictionary<string, string> headers = null, CancellationToken token = default)
+		public async UniTask<HttpResponseObject<T>> Patch<T>(string path, object body = null, Dictionary<string, string> headers = null, CancellationToken token = default)
 		{
 			return await Request<T>(HttpMethod.Patch, path, headers, body, token: token);
 		}
```
Comment (saito-sv) [inline Runtime/ApiClient.cs:125]: Any reason why these are commented out?

Comment (auroter) [reply Runtime/ApiClient.cs:125]: We aren't using them, they are left over from the code we used as a basis for our HTTP stuff.

Comment (auroter) [reply Runtime/ApiClient.cs:125]: Maybe I should just delete them?  I deleted other unused functions.

**Review by ocasta181 — APPROVED** [review]

**Review by saito-sv — APPROVED** [review]

**Pull request merged**